linode_api4/objects/object_storage.py

```diff
@@ -23,19 +23,25 @@ class ObjectStorageACL(StrEnum):
     CUSTOM = "custom"
 
 
+class ObjectStorageKeyPermission(StrEnum):
+    READ_ONLY = "read_only"
+    READ_WRITE = "read_write"
+
+
 class ObjectStorageBucket(DerivedBase):
     """
     A bucket where objects are stored in.
 
     API documentation: https://www.linode.com/docs/api/object-storage/#object-storage-bucket-view
     """
 
-    api_endpoint = "/object-storage/buckets/{cluster}/{label}"
-    parent_id_name = "cluster"
+    api_endpoint = "/object-storage/buckets/{region}/{label}"
+    parent_id_name = "region"
     id_attribute = "label"
 
     properties = {
-        "cluster": Property(identifier=True),
+        "region": Property(identifier=True),
+        "cluster": Property(),
```

Comment:
Should we add a deprecated note to the `cluster` property as well?

Reply:
Thanks for the head up! I think there isn't a clear way to mark it as deprecated in Python since the property is in a dict... Let me think about it maybe we can modify `Property` class to accept a deprecation flag. I can do that in a future PR, I think.